neurodamus/core/coreneuron_configuration.py

```diff
@@ -219,7 +219,7 @@ def write_sim_config(
                 fp.write("'model-stats'\n")
             if enable_reports:
                 fp.write(f"report-conf='{self.output_root}/{self.report_config_file}'\n")
-            fp.write("mpi=true\n")
+            fp.write(f"mpi={os.environ.get('NEURON_INIT_MPI', '1')}\n")
```

Comment:
do you think the various tox tests can be unified on this `neuron_INIT_MPI` now? Because we have some folders (unit) that fix this to 0 and others (scientific-e2e) to 1